keyboards/planck/keymaps/vxid/readme.md

```diff
@@ -0,0 +1,11 @@
+# Planck
```

Comment:
This readme is actually intended specifically for your keymap. What you have here is a readme for the keyboard. In here you can put notes on what your keymap does differently than default, how it's built, any quirks, etc etc.

Reply:
Hi.
My bad, I added a brief description of my keymap.